404.html

```diff
@@ -3,22 +3,28 @@
 
 <head>
   <title>Page not found</title>
-  <script type="importmap">
+  <meta
+    http-equiv="Content-Security-Policy"
+    content="script-src 'nonce-aem' 'strict-dynamic' 'unsafe-inline' http: https:; base-uri 'self'; object-src 'none';"
+    move-to-http-header="true"
+  >
+  <script nonce="aem" type="importmap">
     {
         "imports": {
             "@dropins/tools/": "/scripts/__dropins__/tools/",
             "@dropins/storefront-cart/": "/scripts/__dropins__/storefront-cart/",
-            "@dropins/storefront-auth/": "/scripts/__dropins__/storefront-auth/"
+            "@dropins/storefront-auth/": "/scripts/__dropins__/storefront-auth/",
+            "@dropins/storefront-personalization/": "/scripts/__dropins__/storefront-personalization/"
```

Comment:
This change is not really related to the CSP configuration, the 404 page was already broken and I fixed it to make sure the nonce attributes are correctly placed.